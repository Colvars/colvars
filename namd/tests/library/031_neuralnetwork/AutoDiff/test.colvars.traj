# step         nn_output_1           r_nn_output_1         
           0    4.27348452931304e-01  4.27348452931304e-01 
           1    4.27087400544899e-01  4.27348312414008e-01 
           2    4.26872450296396e-01  4.27346551743906e-01 
           3    4.26690647683911e-01  4.27342076152309e-01 
           4    4.26522005235959e-01  4.27338674601854e-01 
           5    4.26348818295766e-01  4.27335730155590e-01 
           6    4.26238043055371e-01  4.27334079828963e-01 
           7    4.26159193274459e-01  4.27337761524583e-01 
           8    4.26065217585930e-01  4.27338778273542e-01 
           9    4.25982411875574e-01  4.27331353969824e-01 
          10    4.25889928078528e-01  4.27321182692511e-01 
          11    4.25796519562365e-01  4.27310793395096e-01 
          12    4.25853460937317e-01  4.27299903044534e-01 
          13    4.25947005263650e-01  4.27286305999077e-01 
          14    4.26005311535346e-01  4.27270581213223e-01 
          15    4.25997078773281e-01  4.27255905652364e-01 
          16    4.26005236400443e-01  4.27240725046359e-01 
          17    4.26151479874847e-01  4.27224521443315e-01 
          18    4.26359335262689e-01  4.27208896839964e-01 
          19    4.26611349581861e-01  4.27194935615553e-01 
          20    4.26944443559188e-01  4.27183631213753e-01 
