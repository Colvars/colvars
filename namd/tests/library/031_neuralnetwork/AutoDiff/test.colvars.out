colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "nn_output_1"
colvars:   Initializing a new "NeuralNetwork" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # name = "001"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # name = "phi_1"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = off
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 12.011, total charge = 0.51.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 14.007, total charge = -0.47.
colvars:       Initializing atom group "group3".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group3" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:       Initializing atom group "group4".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group4" defined with 1 atoms requested: total mass = 12.011, total charge = 0.51.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:     This colvar uses a custom function.
colvars:     # name = "002"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # name = "phi_2"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = off
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 12.011, total charge = 0.51.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 14.007, total charge = -0.47.
colvars:       Initializing atom group "group3".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group3" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:       Initializing atom group "group4".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group4" defined with 1 atoms requested: total mass = 12.011, total charge = 0.51.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:     This colvar uses a custom function.
colvars:     # name = "003"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # name = "phi_3"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = off
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 12.011, total charge = 0.51.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 14.007, total charge = -0.47.
colvars:       Initializing atom group "group3".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group3" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:       Initializing atom group "group4".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group4" defined with 1 atoms requested: total mass = 12.011, total charge = 0.51.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:     This colvar uses a custom function.
colvars:     # name = "004"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # name = "psi_1"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = off
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 14.007, total charge = -0.47.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:       Initializing atom group "group3".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group3" defined with 1 atoms requested: total mass = 12.011, total charge = 0.51.
colvars:       Initializing atom group "group4".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group4" defined with 1 atoms requested: total mass = 14.007, total charge = -0.47.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:     This colvar uses a custom function.
colvars:     # name = "005"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # name = "psi_2"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = off
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 14.007, total charge = -0.47.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:       Initializing atom group "group3".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group3" defined with 1 atoms requested: total mass = 12.011, total charge = 0.51.
colvars:       Initializing atom group "group4".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group4" defined with 1 atoms requested: total mass = 14.007, total charge = -0.47.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:     This colvar uses a custom function.
colvars:     # name = "006"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # name = "psi_3"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = off
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 14.007, total charge = -0.47.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:       Initializing atom group "group3".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group3" defined with 1 atoms requested: total mass = 12.011, total charge = 0.51.
colvars:       Initializing atom group "group4".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group4" defined with 1 atoms requested: total mass = 14.007, total charge = -0.47.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:     This colvar uses a custom function.
colvars:     # output_component = 0
colvars:     # layer1_WeightsFile = "dense_weights.txt"
colvars:     Will read layer[1] weights from dense_weights.txt
colvars:     # layer2_WeightsFile = "dense_1_weights.txt"
colvars:     Will read layer[2] weights from dense_1_weights.txt
colvars:     # layer1_BiasesFile = "dense_biases.txt"
colvars:     Will read layer[1] biases from dense_biases.txt
colvars:     # layer2_BiasesFile = "dense_1_biases.txt"
colvars:     Will read layer[2] biases from dense_1_biases.txt
colvars:     # layer1_activation = "tanh"
colvars:     The activation function for layer[1] is tanh
colvars:     # layer2_activation = "tanh"
colvars:     The activation function for layer[2] is tanh
colvars:     Warning: cannot cite unknown feature "NeuralNetwork colvar component (derived from neuralNetwork)"
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.01
colvars:   # lowerBoundary = -1
colvars:   # upperBoundary = 1
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = on
colvars:   Enabling the extended Lagrangian term for colvar "nn_output_1".
colvars:   # extendedTemp = 300 [default]
colvars:   # extendedFluctuation = 0.01
colvars:   Computed extended system force constant: 5961.57 [E]/U^2
colvars:   # extendedTimeConstant = 200
colvars:   Computed fictitious mass: 6.04034e+06 [E]/(U/fs)^2   (U: colvar unit)
colvars:   # outputEnergy = off [default]
colvars:   # extendedLangevinDamping = 1 [default]
colvars:   # reflectingLowerBoundary = off [default]
colvars:   # reflectingUpperBoundary = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = on
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { nn_output_1 }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # applyBias = on [default]
colvars:   # updateBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 2000
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 100000
colvars:   # shared = off [default]
colvars:   # maxForce =  [default]
colvars:   # CZARestimator = on [default]
colvars:   # integrate = on [default]
colvars:   # UIestimator = off [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (11 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - BAOA integrator:
colvars:   BouRabee2010 https://doi.org/10.1137/090758842
colvars: 
colvars: - neuralNetwork colvar component:
colvars:   Chen2022 https://doi.org/10.1021/acs.jcim.1c01010
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Optimal rotation via flexible fitting:
colvars: - dihedral colvar component:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - ABF colvar bias implementation:
colvars:   Henin2010 https://doi.org/10.1021/ct9004432
colvars: 
colvars: - CZAR eABF estimator:
colvars: - eABF implementation:
colvars:   Lesage2017 https://doi.org/10.1021/acs.jpcb.6b10055
colvars: 
colvars: - NAMD engine:
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: updating target temperature (T = 300 K).
colvars: Re-initialized atom group for variable "nn_output_1":0/0. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/1. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/2. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/3. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/4. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/5. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/6. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/7. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/8. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/9. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/10. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/11. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/12. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/13. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/14. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/15. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/16. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/17. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/18. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/19. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/20. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/21. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/22. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/23. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: The restart output state file will be "test.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: Updating NAMD interface:
colvars: updating atomic data (11 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 300 K).
colvars: Re-initialized atom group for variable "nn_output_1":0/0. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/1. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/2. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/3. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/4. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/5. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/6. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/7. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/8. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/9. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/10. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/11. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/12. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/13. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/14. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/15. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/16. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/17. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/18. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/19. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/20. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "nn_output_1":0/21. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "nn_output_1":0/22. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "nn_output_1":0/23. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: The restart output state file will be "test.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars:   Prepared sample and gradient buffers at step 0.
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
